internal/scaffold/role.go

```diff
@@ -140,6 +140,9 @@ func UpdateRoleForResource(r *Resource, absProjectPath string) error {
 		}
 		m := &map[string]interface{}{}
 		err = yaml.Unmarshal(d, m)
+		if err != nil {
+			return fmt.Errorf("failed to parse role %q: %v", role.GetName(), err)
```

Comment:
Nit: I actually prefer your added error message format here by a long shot, but it doesn't match any of the other error messages in this function. Maybe use the existing format for consistency for now and then fix all of the formatting in this file in a follow-up PR?

```suggestion
			return fmt.Errorf("failed to unmarshal role(%+v): %v", role, err)
```

Reply:
Since it is too small then, I did the change here in another commit. 
I hope that its fine.